### DISCUSSION TITLE
blog/try-ai-models-locally

### DISCUSSION THREAD

**Discussion by giscus** [Pathsis/2750-words#44] [Announcements]
# blog/try-ai-models-locally

Two thousand seven hundred and fifty words

https://pathos.page/blog/try-ai-models-locally

**Comment by mihouta** [comment]
最近也在思考这个问题，目前问答式AI对话机器人已经有了足够的长下文分析能力，接下来就是根据每个人的情况，提供独有的笔记支持的时候了。反复研究笔记软件多年，很认同你的猜测，不日即将有真正创新的产品出现了。

**Reply by Pathsis** [reply]
共同期待~
